gemfile

```diff
@@ -0,0 +1,35 @@
+
+source 'https://rubygems.org' do
+  gem 'github-pages', group: :jekyll_plugins
+
+   group :jekyll_plugins do
```

Comment:
These dependencies are part of `github-pages`. You don't need to spell them out explicitly.